templates/status.html

```diff
@@ -14,63 +14,85 @@
 {% endblock breadcrumbs %}
 
 {% block content %}
-  <h1 class="text-3xl break-words pt-2 pb-4 md:pt-0 md:pb-6 md:text-4xl font-bold text-slate-900">
-    Status
-  </h1>
-  <div class="grid gap-8 md:grid-cols-2 lg:grid-cols-3 mb-8">
-    {% for backend in backends %}
-      {% #card %}
-        <div class="px-4 py-2 sm:px-6 sm:py-3">
-          <div class="flex flex-wrap items-center justify-between sm:flex-nowrap">
-            <h2 id="workspaces-title" class="text-xl font-semibold tracking-tight text-slate-900">
-              {{ backend.name }}
-            </h2>
-            {% if backend.show_warning %}
-              {% pill variant="warning" text="Missing" %}
-            {% endif %}
+  <div class="prose">
+    <h1>OpenSAFELY Status</h1>
+    <p>This page provides information about the status of the OpenSAFELY platform.</p>
```

Comment:
Thank you for adding extra context on this page. It's needed an update for a while!